package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "open-msupply",
   "//": "Main version for the app, should be in semantic version format (any release candidate or test build should be separated by '-' i.e. 1.1.1-rc1 or 1.1.1-test",
-  "version": "1.1.16",
+  "version": "1.2.01",
```

Comment:
Not too sure what version to use right now.

Reply:
1.2.0 pls